agent/agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -362,8 +389,16 @@ func (m *PGStatMonitorQAN) getSettings() (settings, error) {
 	settings := make(settings)
 	for _, row := range settingsRows {
```

Comment:
if we know len of settingsRows then we could probably initialize map with defined capacity?

Reply:
In case of more settings results in future we will need new commit I think.